connectors/sources/mysql.py

```diff
@@ -435,80 +387,42 @@ async def fetch_documents(self, database, table, query=QUERIES["TABLE_DATA"]):
                     keys_value += f"{row.get(key)}_" if row.get(key) else ""
                 row.update(
                     {
-                        "_id": f"{database}_{table}_{keys_value}",
+                        "_id": f"{self.database}_{table}_{keys_value}",
```

Comment:
If we are planning to move forward with only one database we can remove database for preparing `_id` and same for preparing keys [`keys.append(f"{self.database}_{table}_{column_name[0]}")`](https://github.com/elastic/connectors-python/pull/519/files#diff-996ac0843d7656c71035ae294db22a975939f9fd73685747f5faa8e3b6d64d4eR369)

Reply:
This will be part of future work as the `generic_database_source.py` does the same thing for one database. An upcoming PR will make it consistently across all SQL-like connectors